crates/bevy_pbr/src/wireframe.rs

```diff
@@ -49,31 +67,169 @@ fn extract_wireframe_config(mut commands: Commands, wireframe_config: Res<Wirefr
     }
 }
 
-fn extract_wireframes(mut commands: Commands, query: Query<Entity, With<Wireframe>>) {
-    for entity in query.iter() {
-        commands.get_or_spawn(entity).insert(Wireframe);
+fn extract_wireframes(mut commands: Commands, query: Query<(Entity, Read<Wireframe>)>) {
```

Comment:
```suggestion
fn extract_wireframes(mut commands: Commands, query: Query<(Entity, &Wireframe)>) {
```

Reply:
TIL that this is a type alias... https://docs.rs/bevy/latest/bevy/ecs/system/lifetimeless/type.Read.html

Reply:
I removed it from here, but I kept the one in the lifetimeless variant below (it's a bit less verbose?)